### ISSUE TITLE
Prowlers-n-Paragons wrong Talent name

### ISSUE THREAD

**Issue by CameroC13** [Roll20/roll20-character-sheets#7357]
In roll20 one of the Talents has the title of Riding. This is not an in-game talent. Maybe it existed in an older version. But whatever the case it looks like Investigation is missing from the list and Riding was added in its place. I pasted a screenshot of the character sheet taken from the Waiting Room Edition of the Book. 

Hopefully this is an easy fix. Thank you

![image](https://user-images.githubusercontent.com/71046294/92662834-910ffa80-f2b4-11ea-885f-455595f22d00.png)

**Comment by CameroC13** [comment]
I was wrong. There are actually more errors. The sheet is missing Agility and Intellect as well. See the sheet in Roll20 below
![image](https://user-images.githubusercontent.com/71046294/94183729-bde91380-fe57-11ea-9b57-317534b3852a.png)

**Comment by nmbradley** [comment]
@finderski this appears to be one of your sheets; is this something you're potentially willing to explore and fix?

**Comment by finderski** [comment]
@nmbradley unfortunately, this is not a game I play and did the sheet waaaay back when as a favor. I made the requested changes, though really the sheet needs a complete overhaul. Unfortunately, I don't have time, nor the desire to make such an overhaul.  However, I did add Agility, Covert, Intellect, and Investigation to the existing sheet.

This will be my last update for this sheet, however.  Sorry about that, I just don't have time to support a sheet for a game I have never and will never play. :(
